src/Application/AbstractMagpieCommand.php

```diff
@@ -2,26 +2,41 @@
 
 namespace Ravenfire\Magpie\Application;
 
+use Ravenfire\Magpie\Application\SqlScripts\SqlTraits;
 use Ravenfire\Magpie\Magpie;
 use Symfony\Bridge\Monolog\Handler\ConsoleHandler;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * Lays foundation for the command files.
+ */
 class AbstractMagpieCommand extends Command
 {
+    use SqlTraits;
```

Comment:
I would probably not add this to `AbstractMagpieCommand` since not everything that extends this class actually needs to use the sql stuff. I'd add this to each of the Sql script commands instead, and document somewhere "when making a Sql command, you can use the `SqlTraits`.

Reply:
Also, its minor, but traits are usually singular and are often names as if its a trait that something can have like:

`CanParseSqlResults` or `CanHandleSql`. But not always `SqlTrait` is perfectly fine.